src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -234,6 +219,17 @@ private String getEIBackendSubscriptionAddress() {
         return httpMethod + "://" + this.getBackendServerHost() + ":" + this.getBackendServerPort();
     }
 
+    private String getEIRequestURL(HttpServletRequest request) {
+        String eiBackendAddressSuffix = request.getServletPath();
+        String requestQuery = request.getQueryString();
+        String query = (requestQuery != null && !requestQuery.isEmpty()) ? "?" + requestQuery : "";
+        String requestUrl = getEIBackendSubscriptionAddress() + eiBackendAddressSuffix + query;
```

Comment:
Can't "eiBackendAddressSuffix" variable be moved into getEIBackendSubscriptionAddress() function?